packages/cloud_functions/cloud_functions/pubspec.yaml

```diff
@@ -20,12 +20,9 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  flutter_driver:
-    sdk: flutter
   flutter_test:
     sdk: flutter
   plugin_platform_interface: ^2.0.0
-  test: any
```

Comment:
I removed the `test` dependencies in all packages (except `_flutterfire_internals`) because the packages already have `flutter_test` as a dependency. Or is there a reason why we have two test dependencies?